styleguide/_themes/derek/scss/patterns/grid/productFeatures.scss

```diff
@@ -30,11 +31,11 @@
 }
 
 @media only screen and (max-width: $screen-sm) {
-  .features {
+  .features-container {
     display: block;
     margin-top: 10px;
 
-    .desc {
+    .features-desc {
```

Comment:
Yeah it's a bug fix and I snuck in some naming changes. I don't want to go full refactor on a super quick task for right now if that's fine with everyone.